src/components/Fetch/Bookmarks.js

```diff
@@ -45,12 +49,14 @@ export class Bookmarks extends React.PureComponent<Props> {
 export type BookmarkProps = {
   lineNumber: number,
   scrollFunc: (event: SyntheticMouseEvent<HTMLInputElement>) => void,
+  emphasize: boolean,
```

Comment:
I kind of feel like the arrow icon isn't needed. It's not really persistent (it doesn't do anything after the initial load) and the icon would add to visual clutter slightly. What do you think?

Reply:
currently the arrow in the bookmarks panel is useful to the person creating the "shareline" so they know which line# they have selected to share with their peer and which line# is a bookmark. 
if the bookmarks and shareline were separated in 2 vertical columns then we could omit the the arrow icon

Reply:
My point is that the user doesn't need to know the position of the shareline, because it's not relevant to their current workflow. I think Github's share line feature is a good analogue - you can bookmark any line for any reason which adds a visual indicator. You can separately copy a permalink but that neither adds a visual indicator nor does it interact with the bookmark